mmv1/third_party/terraform/services/bigtable/resource_bigtable_table_test.go

```diff
@@ -415,6 +415,27 @@ func TestAccBigtableTable_automated_backups(t *testing.T) {
 				ImportStateVerify:       true,
 				ImportStateVerifyIgnore: []string{"deletion_protection", "instance_type"},
 			},
+			// Creating a table with automated backup explicitly disabled
```

Comment:
This should be a separate test function. We don't typically create --> destroy --> create --> destroy within a single test. It should be two separate tests that create --> update (if applicable) --> destroy